ci/pipeline.yml

```
@@ -35,197 +25,172 @@ jobs:
-      file: bosh-openstack-cpi-release/ci/tasks/terraform-destroy-lifecycle.yml
-      params: *lifecycle-terraform-params
+      AUTH_URL: http://10.100.30.2/identity
+      OS_USERNAME: ((openstack_ci_user.username))
+      OS_PASSWORD: ((openstack_ci_user.password))
+      OS_PROJECT: bosh
 
-- name: bats-ubuntu-manual
+- name: run-devstack-tests
   serial: true
-  serial_groups: [ cleanup-lock-bats-ubuntu-manual ]
+  serial_groups: [ devstack ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)